releases/release-6.0.0-dmr.md

```diff
@@ -0,0 +1,586 @@
+
+    [User document](/placement-rules-in-sql.md)
+
+- Support building TiFlash replicas by databases. To add TiFlash replicas for all tables in a database, you only need to use a single SQL statement, which greatly saves operation and maintenance costs.
+
+    [User document](/tiflash/use-tiflash.md#create-tiflash-replicas-for-databases)
+
+### Transaction
+
+- Add a check for data index consistency at the kernel level
+
+    Add a check for data index consistency when a transaction is executed, which improves system stability and robustness, with only very low resource overhead. You can control the check behavior using the `tidb_enable_mutation_checker` and `tidb_txn_assertion_level` variables. With the default configuration, the QPS drop is controlled within 2% in most scenarios. For the error description of the consistency check, see [user document](/data-inconsistency-errors.md).
+
+### Observability
+
+- ​​General availability (GA) of Top SQL
+
+    Top SQL is a self-service database performance monitoring and diagnosis feature provided in TiDB Dashboard for DBAs and app developers. With this feature, you can easily locate SQL queries that contribute to a high load of a TiDB or TiKV node in a specified time range. Unlike existing diagnostic features provided in TiDB Dashboard for database experts, Top SQL is designed for non-experts. You do not need to traverse thousands of monitoring charts to find correlations or understand TiDB internal mechanisms such as Raft Snapshot, RocksDB, MVCC, and TSO. Understanding basic concepts, such as index, lock conflict, and execution plans, is enough for you to use Top SQL to analyze database load quickly and improve application performance.
+
+    Top SQL is disabled by default and can be enabled with a single click. When enabled, Top SQL provides you with the CPU load of each TiKV or TiFlash node within 30 days. Therefore, you can spot SQL statements consuming high CPU loads at first glimpse, and quickly analyze the issues such as database hotspots and sudden load increases. For example, you can use Top SQL to locate an analytical query that consumes 99% of the load for a low-load database.
+
+    [User documentation](/dashboard/top-sql.md)
+
+- GA of Continuous Profiling
+
+    TiDB Dashboard introduces the Continuous Profiling feature, providing the ability to automatically save instance performance analysis results as the cluster is running, improving the observability of TiDB cluster performance and helping to reduce troubleshooting time.
```

Comment:
```suggestion
    TiDB Dashboard introduces the Continuous Profiling feature since v5.4.0, which is now generally available in v6.0.0. When enabled, the performance data of each TiDB, TiKV and PD instances will be collected all the time, with negligible overhead. These history performance data can be used for technical experts to backtrack and pinpoint the root cause of production issues like high memory consumption, even when they are hard to replicate.
```